flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -157,6 +168,12 @@ public Builder streaming(boolean streaming) {
       return this;
     }
 
+    public Builder exposeLocality(boolean newExposeLocality) {
+      contextBuilder.exposeLocality(newExposeLocality);
```

Comment:
we set exposeLocality to contextBuilder in the `buildFormat` method. we probably don't need to do it here

Reply:
removed